phi/tools/function.py

```diff
@@ -175,7 +175,7 @@ def process_entrypoint(self, strict: bool = False):
         except Exception as e:
             logger.warning(f"Could not parse args for {self.name}: {e}", exc_info=True)
 
-        self.description = getdoc(self.entrypoint)
+        self.description = getdoc(self.entrypoint) or self.description
```

Comment:
So the entrypoint doc should overwrite it?

Reply:
If the function has docstring it's good else go with the description